### DISCUSSION TITLE
website sidebar home URL: how to set it as "index.html"

### DISCUSSION THREAD

**Discussion by aborruso** [quarto-dev/quarto-cli#10053] [Q&A] [answered]
### Description

Hi,
I have a site that contains only a sidebar e no navbar.

When I put my mouse over the home ("Lorem") in the preview link it points to `/` and not to `/index.html`

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/e5f4b40c-9ad6-481b-a76e-33b1ba1794b5)

Is there a way to force to have  `/index.html`, to avoid to open the folder that contain the website, when I open the website in local without using a webserver?

I'm using quarto 1.4.551 in debian 12.5.

Thank you

![image](https://github.com/quarto-dev/quarto-cli/assets/30607/425f27b1-3c15-4fa3-8daa-fb90a8d18fc6)

**Comment by mcanouil** [comment, marked as answer]
I believe your are looking for `logo-href`, see <https://quarto.org/docs/reference/projects/websites.html#navbar>.

Edit: I thought the sidebar also had logo-href but that's in the pre-release, see <https://prerelease.quarto.org/docs/reference/projects/websites.html#sidebar>.

**Reply by aborruso** [reply]
@mcanouil I had already seen the navbar, but I don't have it. And in fact I saw that you edited your answer.

As usual thank you for your fast and detailed response. For the time being I can't use the pre-release

I will do a find and replace in the output HTML.
